2020/7xxx/CVE-2020-7377.json

```diff
@@ -1,18 +1,105 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "cve@rapid7.com",
+        "DATE_PUBLIC": "2020-08-18T08:48:00.000Z",
         "ID": "CVE-2020-7377",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Rapid7 Metasploit Framework Relative Path Traversal in telpho10_credential_dump module"
     },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Metasploit Framework",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": ">=",
+                                            "version_name": "4.12.40",
+                                            "version_value": "4.12.40"
+                                        },
+                                        {
+                                            "version_affected": "<",
+                                            "version_name": "6.0.2",
+                                            "version_value": "6.0.2"
```

Comment:
This issue has not yet been fixed. Presuming that my open PR (https://github.com/rapid7/metasploit-framework/pull/14034) is reviewed, found to be sufficient, and merged this week, the versions affected would be `< 6.0.3`.

Reply:
Ah, thanks. Will update both!

Reply:
No need to update both.

CVE-2020-7376 was resolved in 6.0.2.

CVE-2020-7377 has now been patched and the patch will be available in the 6.0.3 release this week.